daemon/cluster/executor/container/container.go

```diff
@@ -124,7 +124,7 @@ func (c *containerConfig) labels() map[string]string {
 		system = map[string]string{
 			"task":         "", // mark as cluster task
 			"task.id":      c.task.ID,
-			"task.name":    fmt.Sprintf("%v.%v", c.task.ServiceAnnotations.Name, c.task.Slot),
```

Comment:
You need to handle fallback here. if `c.task.Annotations.Name` is not set, fallback to this `fmt.Sprintf` value.